### DISCUSSION TITLE
Any interest in a grid style meow-cheatsheet-physical-layout?

### DISCUSSION THREAD

**Discussion by willbush** [meow-edit/meow#626] [Q&A] [answered]
I setup my own cheatsheet physical layout [here](https://github.com/willbush/system/blob/e9f86c035c822f37109e706022e690ad95275cf6/configs/emacs/meow/modules/init-meow.el#L4) because I use a [planck](https://olkb.com/collections/planck). I also use a [corne keyboard](https://github.com/foostan/crkbd), but probably not going through the effort to make that layout.

Any interest in me submitting a PR? I could make it more universal. perhaps a 4x12 grid and 5x12 grid (like the peronic) that should cover most grid style keyboards. I also think people using column staggered keyboards would prefer to see a grid cheat-sheet instead of row staggered.

**Comment by eshrh** [comment, marked as answer]
Sure. I'd go for a 4-row grid so you can match the number of keys on an ISO/ANSI keyboard. Keyboard-specific layouts are probably a bit too far, but adding an ortholinear option is fun. I use a corne as well!

**Reply by willbush** [reply]
> I use a corne as well!

Ah sweet! There's dozens of us :)

**Reply by willbush** [reply]
@eshrh wait you use NixOS too?! Emacs meow corne nixos.. who knows what else? Astronomical odds.. never tell me the odds!

**Reply by eshrh** [reply]
haha yeah! Although, i think you might be surprised at how popular nixos and the corne keyboard are getting these days. Emacs though, that does seem to be a bit rarer...
